### PR TITLE
Update INSTALLATION.md

### PR DESCRIPTION
**Pull request by kushal34712**
Made some small change will enhance the quilty of reading

### CODE REVIEW

**Review by jrobinAV — APPROVED** [review]

File: INSTALLATION.md [hunk 1]
```diff
@@ -109,17 +109,17 @@ TypeScript code from your debugger.
 >
 > When you are developing front-end code for the Taipy GUI package, it may
 > be cumbersome to have to install the package over and over when you know
-> that all that has changed is the JavaScript bundle that makes the Taipy
+> That all that has changed is the JavaScript bundle that makes the Taipy
```
Comment (trgiangdo) [inline INSTALLATION.md:112]: This is not a new sentence. Capitalize here doesn't seem right

File: INSTALLATION.md [hunk 2]
```diff
@@ -109,17 +109,17 @@ TypeScript code from your debugger.
...
 > You can, however, set the environment variable `TAIPY_GUI_WEBAPP_PATH`
 > to the location of your choice, and Taipy GUI will look for the web
-> app in that directory.<br/>
-> If you set this variable to the location where you build the web app
-> repeatedly, you will no longer have to reinstall Taipy GUI before you
-> try your code again.
+> App in that directory.<br/>
+> If you set this variable to the location where you build the web app.
+> Repeatedly, you will no longer have to reinstall Taipy GUI before you.
+> Try your code again.
```
Comment (trgiangdo) [inline INSTALLATION.md:122]: These are all in the same sentence. Capitalize here doesn't seem right

**Review by trgiangdo — CHANGES_REQUESTED** [review]

**Review by kushal34712 — COMMENTED** [review]
i have made  changes

**Comment by trgiangdo** [comment]
> i have made changes

I don't see any new commit. Did I miss something by any chance?

**Comment by kushal34712** [comment]
i have made changes but its not reflecting in the same pr do not know why

**Pull request closed without merging**